VenafiPS/Public/Get-TppVersion.ps1

```diff
@@ -47,8 +47,7 @@ function Get-TppVersion {
     }
 
     try {
-        $ver = [Version]((Invoke-TppRestMethod @params).Version)
-        [Version]::new($ver.Major, $ver.Minor, $ver.Build)
+        [Version]((Invoke-TppRestMethod @params).Version)
```

Comment:
Why not change this to 'Invoke-VenafiRestMethod' while we're changing this?

Reply:
no reason not to, done.